tests/creusot-contracts/creusot-contracts/why3session.xml

```diff
@@ -560,41 +750,49 @@
 </theory>
 <theory name="M_creusot_contracts__stdqy35z1__cmp__qyi8793870187857001919__refl" proved="true">
  <goal name="vc_refl&#39;0" proved="true">
+ <proof prover="0"><result status="valid" time="0.018866" steps="2896"/></proof>
```

Comment:
These are duplicate proofs. Are you manually selecting Z3 in the provers? Strategies like "auto level 1" (shortcut key 1) clean up the tree when multiple provers succeed.